SafeExamBrowser.UserInterface.Desktop/Windows/FileSystemDialog.xaml.cs

```diff
@@ -286,21 +288,50 @@ private void InitializeDialog()
 			InitializeText();
 			InitializeFileSystem();
 		}
-		
+
 		private DriveInfo[] GetDrives(bool showAll = false)
```

Comment:
The `showAll` flag isn't really used, correct? If so, I'd remove it to simplify the logic.

Reply:
Ok I will edit the commit during this week. What I was wondering about the mobile UI part is why the code is not swapped out to a ViewModel so it is less redundant?

Reply:
Excellent observation. I had to make a decision when designing the architecture, namely whether the user interfaces should share their code or whether they should be completely independent. The former leads to coupling (i.e. changes in one UI affect all the others), the latter leads to code duplication. As I wanted to be able to make a UI that is radically different from others, I chose the latter.